packages/camera/camera_windows/lib/camera_windows.dart

```diff
@@ -0,0 +1,437 @@
+
+    /// Creates channel for camera events.
+    _cameraChannels.putIfAbsent(requestedCameraId, () {
+      final MethodChannel channel =
+          MethodChannel('flutter.io/cameraPlugin/camera$requestedCameraId');
+      channel.setMethodCallHandler(
+        (MethodCall call) => handleCameraMethodCall(call, requestedCameraId),
+      );
+      return channel;
+    });
+
+    final Map<String, double>? reply;
+    try {
+      reply = await _pluginChannel.invokeMapMethod<String, double>(
+        'initialize',
+        <String, dynamic>{
+          'cameraId': requestedCameraId,
+        },
+      );
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+
+    cameraEventStreamController.add(
+      CameraInitializedEvent(
+        requestedCameraId,
+        reply!['previewWidth']!,
+        reply['previewHeight']!,
+        ExposureMode.auto,
+        false,
+        FocusMode.auto,
+        false,
+      ),
+    );
+  }
+
+  @override
+  Future<void> dispose(int cameraId) async {
+    if (_cameraChannels.containsKey(cameraId)) {
+      final MethodChannel? cameraChannel = _cameraChannels[cameraId];
+      cameraChannel?.setMethodCallHandler(null);
+      _cameraChannels.remove(cameraId);
+    }
+
+    try {
+      await _pluginChannel.invokeMethod<void>(
+        'dispose',
+        <String, dynamic>{'cameraId': cameraId},
+      );
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  @override
+  Stream<CameraInitializedEvent> onCameraInitialized(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraInitializedEvent>();
+  }
+
+  @override
+  Stream<CameraResolutionChangedEvent> onCameraResolutionChanged(int cameraId) {
+    /// Windows camera plugin does not support resolution changed events.
+    return const Stream<CameraResolutionChangedEvent>.empty();
+  }
+
+  @override
+  Stream<CameraClosingEvent> onCameraClosing(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraClosingEvent>();
+  }
+
+  @override
+  Stream<CameraErrorEvent> onCameraError(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraErrorEvent>();
+  }
+
+  @override
+  Stream<VideoRecordedEvent> onVideoRecordedEvent(int cameraId) {
+    return _cameraEvents(cameraId).whereType<VideoRecordedEvent>();
+  }
+
+  @override
+  Stream<DeviceOrientationChangedEvent> onDeviceOrientationChanged() {
+    /// Windows camera plugin does not support capture orientations.
+    /// Force device orientation to landscape as by default camera plugin uses portraitUp orientation.
+    return Stream<DeviceOrientationChangedEvent>.value(
+      const DeviceOrientationChangedEvent(DeviceOrientation.landscapeRight),
+    );
+  }
+
+  @override
+  Future<void> lockCaptureOrientation(
+    int cameraId,
+    DeviceOrientation orientation,
+  ) async {
+    throw UnimplementedError('lockCaptureOrientation() is not implemented.');
```

Comment:
Here and elsewhere: file a bug to implement this and add a TODO.

Alternatively, if this will never be supported, use `UnsupportedError` instead.

https://api.flutter.dev/flutter/dart-core/UnsupportedError-class.html